.pre-commit-config.yaml

```
@@ -63,6 +63,6 @@ repos:
         name: mypy
         language: system
         verbose: true
-        entry: bash -c 'poetry run mypy "azimuth" || true' --
+        entry: bash -c 'poetry run mypy "azimuth"'
```

Without the `||`, we don't need `bash` anymore.
```suggestion
        entry: poetry run mypy azimuth
```